### PR TITLE
test for ijit

### LINKED ISSUE #395: [No Issue]
🎯 **Goal**

We can include an easter egg in the code somewhere!

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by ritzvik**
Fixes #395

### CODE REVIEW

**Comment by ritzvik** [comment]
Well, does this look good?
newplot0
newplot1
newplot2
newplot3

These are fractals from Julia set parametrized by some angle. The angles can be adjusted with a slider.

**Comment by ritzvik** [comment]
@shreyasbapat can you help with the tests?

Also to plot it
```
from einsteinpy.plotting import fractal
from plotly.offline import init_notebook_mode
init_notebook_mode(connected=True)

fractal()
```

**Comment by shreyasbapat** [comment]
I'll have a look at this in some minutes.

**Comment by ritzvik** [comment]
> I'll have a look at this in some minutes.

Well, I wrote the tests! It was good to learn how `mock` works.
Anyway, if you find this good enough, we can merge!

File: src/einsteinpy/plotting/fractal.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
+import numpy as np
```
Comment (shreyasbapat) [inline src/einsteinpy/plotting/fractal.py]: Please mention everything on the top of this module about what it is. For future. Why we added it

**Pull request merged**